cpp/bench/ann/src/raft/raft_benchmark.cu

```diff
@@ -182,6 +188,37 @@ void parse_search_param(const nlohmann::json& conf,
 }
 #endif
 
+#ifdef RAFT_ANN_BENCH_USE_RAFT_CAGRA_HNSWLIB
+template <typename T, typename IdxT>
+void parse_build_param(const nlohmann::json& conf,
+                       typename raft::bench::ann::RaftCagraHnswlib<T, IdxT>::BuildParam& param)
+{
+  if (conf.contains("graph_degree")) {
```

Comment:
Since these are all CAGRA-specific, can we just use the existing CAGRA version? This is going to increase maintenance burden.